### ISSUE TITLE
iOS live editing app crash on js files

### ISSUE THREAD

**Issue by rborn** [NativeScript/NativeScript#4139]
Trying to edit a **js** file on iOS will crash the app on the first save. If I simply save the file again the app will restart ok. So each edit needs 2 save actions.

Full description [here](https://github.com/rborn/nativescript-i18n/issues/47#issuecomment-299447546)

**Comment by b4rtt** [comment]
Same problem here

**Comment by dtopuzov** [comment]
I think this is duplicate of https://github.com/NativeScript/nativescript-cli/issues/2732
We are aware and issue will be fixed in first hot-fix release after 3.0.0 (hope it will be live by the end of this week).

**Comment by atameet** [comment]
I may be wrong here, but looks like this issue is different to #2732.
I have seen a double save issue where one save crashes the ios app in the simulator but keeps the livesync on and another save brings the app back. This has an error:
CoreSimulatorBridge[2296]: Non-FBS error launching.
While another error, which #2732, is where livesync itself crashes on save and we need to run the command again. Here, the error is:
xcrun simctl launch
The following may be completely not related, but one difference I noticed was the prior error in a javascript app and the latter one in a ts app. Could be something else too.

**Comment by dtopuzov** [comment]
Hi @atameet @b4rtt @rborn fix for is already available in master, can you please try:
```
npm un -g nativescript
npm cache clean
npm i -g nativescript@next
```

I'm reopening the issue now, if you still experience problems when save only once, please post more details:
- Can you reproduce it with hello-world app?
- What plugins you use in your app (preferably post our package.json)?
- What is the IDE you use for development?

**Comment by atameet** [comment]
the double refresh issue seems to be working now. thanks.
about the livesync crashing: that too does not seem to crash. 
one thing though. in a livesync with attached debugger is it suppose to re-attach on its own after a file change or is it manual?
thanks

**Comment by dtopuzov** [comment]
Closing since original issue is fixed.
@atameet If you encounter other problems, please log them as separate issues (and provide as much info as possible).

Note that issues related with command-line tooling should be reported here: https://github.com/NativeScript/nativescript-cli

VSCode extension issues should be reported here: 
https://github.com/NativeScript/nativescript-vscode-extension
